exifread/heic.py

```diff
@@ -49,6 +49,9 @@ def get(self, nbytes):
         read = self.file_handle.read(nbytes)
         if not read:
             raise EOFError
+        if len(read) != nbytes:
+            print(f'current pos={self.file_handle.tell()}')
+            raise ValueError(f'get(nbytes={nbytes}) found {len(read)} bytes')
```

Comment:
```suggestion
            msg = "get(nbytes={nbytes}) found {read} bytes at postion {pos}".format(
                nbytes=nbytes,
                read=len(read),
                pos=self.file_handle.tell()
            )
            raise BadSize(msg)
```

Reply:
The f-string format was introduced in Python 3.6 which is why there is a syntax error on previous versions.
Using the `BadSize` exception specific to this class. The exception should get printed by the exception handling mechanism so no need to have a separate print statement.